src/int_set.jl

```diff
@@ -26,9 +26,9 @@ sizehint!(s::IntSet, n::Integer) = (_resize0!(s.bits, n+1); s)
 
 # only required on 0.3:
 function first(itr::IntSet)
-    state = start(itr)
-    done(itr, state) && throw(ArgumentError("collection must be non-empty"))
-    next(itr, state)[1]
+    state = iterate(itr)
```

Comment:
This method is causing an error and should just be removed since there is a fallback that will work here.

Reply:
Yes, I have a fix for this locally